examples/getting_started/2_Pipeline.ipynb

```diff
@@ -145,34 +145,45 @@
+    "One you have determined your mapping, you'll next need to choose a reduction operator to use when aggregating multiple datapoints into a given pixel. For points, each datapoint is mapped into a single pixel, while the other glyphs have spatial extent and can thus map into multiple pixels, each of which operates the same way. All glyphs act like points if the entire glyph is contained within that pixel. Here we will talk only about \"datapoints\" for simplicity, which for an area-based glyph should be interpreted as \"the part of that glyph that falls into this pixel\".\n",
+    "\n",
+    "All of the currently supported reduction operators are incremental, which means that we can efficiently process datasets in a single pass. Given an aggregate bin to update (typically corresponding to one eventual pixel) and a new datapoint, the reduction operator updates the state of the bin in some way. (Actually, datapoints are normally processed in batches for efficiency, but it's simplest to think about the operator as being applied per data point, and the mathematical result should be the same.) A large number of useful [reduction operators]((https://datashader.org/api.html#reductions) are supplied in `ds.reductions`, including:\n",
     "\n",
     "**`count(column=None)`**:\n",
-    "  increment an integer count each time a datapoint maps to this bin.\n",
+    "  increment an integer count each time a datapoint maps to this bin. The resulting aggregate array will be an unsigned integer type, allowing counts to be distinguished from the other types that are normally floating point.\n",
     "  \n",
     "**`any(column=None)`**:\n",
     "  the bin is set to 1 if any datapoint maps to it, and 0 otherwise.\n",
     "  \n",
     "**`sum(column)`**:\n",
     "  add the value of the given column for this datapoint to a running total for this bin.\n",
     "   \n",
-    "**`count_cat(column)`**:\n",
-    "  given a bin with categorical data (i.e., [Pandas' `categorical` datatype](https://pandas-docs.github.io/pandas-docs-travis/categorical.html)), count each category separately, adding the given datapoint to an appropriate category within this bin.  These categories can later be collapsed into a single count if needed; see example below.\n",
+    "**`by(column)`**:\n",
```

Comment:
The signature for `by` is different, not sure how best to express `by(column, agg)`.

Reply:
Good point; need to update that too.